### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by ted-marozzi**

### CODE REVIEW

File: src/propertyvalueestimate.py [hunk 1]
```diff
@@ -5,13 +5,11 @@
 from selenium.webdriver.support import expected_conditions as EC
 from bs4 import BeautifulSoup
 import domainapi as api
+from domainapi import DOMAIN_CLIENT_URL
 
-advertId = 2017019671
-DOMAIN_CLIENT_URL = "https://www.domain.com.au/"
```
Comment (ted-marozzi) [inline src/propertyvalueestimate.py:10]: Moved this to the central domain api file to keep the urls together, note how its imported on the right line 8

File: src/propertyvalueestimate.py [hunk 2]
```diff
@@ -5,13 +5,11 @@
...
-AD_ENDPOINT = DOMAIN_CLIENT_URL + str(advertId)
```
Comment (ted-marozzi) [inline src/propertyvalueestimate.py:11]: This isnt a constant so shouldn't be capital, it should be generated within the funciton, see how i did it

**Review by ted-marozzi — COMMENTED** [review]
I put everything in source so the pylance linter wouldn't complain about missing imports, i found that [here](https://github.com/microsoft/pylance-release/issues/52#issuecomment-653745462)

**Review by lucas-shugg — COMMENTED** [review]
ahh alright yeah i see what you've done, thanks noobos

**Pull request merged**